### PR TITLE
Project Weather App - Johanna Rexin

### PR DESCRIPTION
**Pull request by johannarexin**

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    city.innerHTML = weatherObject.name;
+    weatherDescription.innerHTML = weatherObject.weather[0].description.toUpperCase()
+    temperatures.innerHTML = weatherObject.main.temp.toFixed(0.5)
+    
+    // I've tried to solve the sunrise/sunset to show hhh:mm but didn't suceed. 
+    // Have googled, asked in group chat and read every question I could find on Stack without understanding.
+    // Would very much like to get some help to understand this one.
+
+    // const sunriseValue = weatherObject.sys.sunrise; //Sunrise and Sunset times in UNIX
+    // const sunsetValue = weatherObject.sys.sunset;
```
Comment (moablomkvist) [inline code/script.js:28]: Maybe you can try to make a function of it instead.
I made a function of sunrise and a function of sunset. Like this:  
```suggestion
  const weatherSunrise = () => {
  const dateSunrise = new Date(json.sys.sunrise * 1000);
  const timeSunrise = dateSunrise.toLocaleTimeString([], {
    hour: '2-digit',
    minute: '2-digit',
  });
  document.getElementById('sunrise').innerHTML = timeSunrise;
};
weatherSunrise();
```

File: code/script.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+// Don't know why I can't get the forecast "apifiveDaysForecast" to work. 
+// I believe I have done it in the exactly same way as with "apiStockholm".
+
+// fetch(apiFiveDaysForecast)
+//     .then((response) => {
+//         return response.json();
+// })
+
+// // Five days forecast
+// .then((forecast) => {
```
Comment (moablomkvist) [inline code/script.js:46]: We got this code in the brief: 
const filteredWeek = json.list.filter(item => item.dt_txt.includes('12:00'));

Use it for create the filtered list from the API and then use forEach to modify temperature and weekdays. :)

File: code/script.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+//     fiveDays.innerHTML = forecast.weather[0].id;
+// })
```
Comment (moablomkvist) [inline code/script.js:48]: You can do this!! 💪

File: code/style.css [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    background-color: black;
+    opacity: 0.3;
+    width: fit-content;
+    padding: 10px;
+    color: white;
+    border-radius: 20px;
+}
+
+
+@media (min-width: 668px) and (max-width: 1024px) {
```
Comment (moablomkvist) [inline code/style.css:33]: Good! Responsive <3

File: code/style.css [hunk 5]
```diff
@@ -0,0 +1,86 @@
+body {
+    margin: 0px auto;
+    font-family: Arial, Helvetica, sans-serif;
+    background-image: url("images/background_image.jpg");
```
Comment (moablomkvist) [inline code/style.css:4]: Really like your background-image!

**Review by moablomkvist — CHANGES_REQUESTED** [review]

File: code/index.html [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    <head>
+        <meta charset="UTF-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Johannas Weather Services</title>
+        <link rel="stylesheet" href="./style.css"/>     
+    </head>
+    <body>
+        <section class="weather-section-box">
+        <section class="weather-section">
+            <h1>Today's weather in:</h1>
```
[the remark this answers is not in the document]
Comment (HIPPIEKICK) [reply code/index.html:12]: That's a really good metaphor!

File: code/script.js (hunk 1 above)
Comment (HIPPIEKICK) [reply code/script.js:28]: If you search on sunset in Stack Overflow there are some questions about this as well, if you want to read more :)

File: code/script.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
+const apiKey = '380f07f94efb48727a6ae5c8f7d15f5d';
+const apiStockholm = 'https://api.openweathermap.org/data/2.5/weather?q=Stockholm,Sweden&units=metric&APPID=380f07f94efb48727a6ae5c8f7d15f5d';
+const apiFiveDaysForecast = 'https://api.openweathermap.org/data/2.5/forecast?q=Stockholm,Sweden&units=metric&APPID=380f07f94efb48727a6ae5c8f7d15f5d';
```
Comment (HIPPIEKICK) [inline code/script.js:3]: You have the API-key as a variable, so you can use it inside your other strings using template literals:
`https://api.openweathermap.org/data/2.5/weather?q=Stockholm,Sweden&units=metric&APPID=${apiKey}`

File: code/script.js (hunk 2 above)
Comment (HIPPIEKICK) [reply code/script.js:46]: A good idea is to begin with logging the data to see how it comes back from the API. Start with `console.log(forecast)` to see how this JSON response is different from the first fetch you did. As Moa mentioned all the forecast data is in an array called `list` this time.

**Comment by johannarexin** [comment]
Hello, now I have updated my code and pushed a new version :) 
Best and good night, Johanna

**Comment by HIPPIEKICK** [comment]
> Hello, now I have updated my code and pushed a new version :)
> Best and good night, Johanna

Looks really good Johanna, great job 💪 🥳

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**